Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Button.css';
+
+const ActButton = <button>{}</button>;
+const OpButton = <button>{}</button>;
+
+const ActionButton = props => {
+    return (
+        <div>
+            <button className="action-button">{ActButton.props.text}</button>
+            <button className="operator-button">{OpButton.props.text}</button>
+        </div>
```

Comment:
You can take out this entire block of code and replace it with this:
```
<button className={props.class}>{props.text}</button>
```

The `<div>` will create a new row, but we do not want that (remember that the `ActionButton` component should be for an individual button, not for rows of buttons).

Therefore, all we have to return is a single `<button>` element using the props needed for its styling (`className={props.class}`) and text (`{props.text}`).